main.tf

```diff
@@ -155,6 +155,8 @@ EOF
 resource "aws_vpc_dhcp_options" "dns_resolver" {
   domain_name_servers = [
     "${aws_directory_service_directory.ad.dns_ip_addresses}",
```

Comment:
https://www.terraform.io/docs/configuration/interpolation.html#concat-list1-list2-
join the two lists rather than naming google_dns1&2
if someone provided a map that wasn't google_dns this wouldn't work
see below comment about using a list for public dns